### DISCUSSION TITLE
How to import a swift file from a native module in a native part of the app?

### DISCUSSION THREAD

**Discussion by vladamx** [callstack/react-native-builder-bob#349] [Q&A]
I have a native module as a pod called react-native-carplay.

That native module needs some classes to be passed in from native part of the app.

In order to invoke methods that set those classes i need to import a class from react-native-carplay via import react_native_carplay

Once i do that i get the error

<img width="522" alt="image" src="https://user-images.githubusercontent.com/11381171/211559685-dabd066e-e317-47d3-8f38-e66ba3718d45.png">

**Comment by marlene89** [comment]
Were you able to find a solution for this. I'm facing the same issue on android.

**Reply by vladamx** [reply]
I decided to make a static lib containing swift interface which acts as a bridge between native part of the app and native module so i do not depend on a pod directly but rather to the interface
